framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -113,6 +121,8 @@
     public CFAbstractTransfer(CFAbstractAnalysis<V, S, T> analysis) {
         this.analysis = analysis;
         this.sequentialSemantics = !analysis.checker.hasOption("concurrentSemantics");
+        inferSignatures = analysis.getTypeFactory().getProcessingEnv().
```

Comment:
I would add "this." for consistency with the previous line (or remove "this." from the previous line).